java/src/test/java/ai/rapids/cudf/HostMemoryBufferTest.java

```diff
@@ -40,6 +43,19 @@ void testRefCountLeak() throws InterruptedException {
     assertEquals(expectedLeakCount, MemoryCleaner.leakCount.get());
   }
 
+  @Test
+  void asByteBuffer() {
+    final long size = 1024;
+    try (HostMemoryBuffer buff = HostMemoryBuffer.allocate(size)) {
+      ByteBuffer dbuff = buff.asByteBuffer();
```

Comment:
Recommend verifying reported buffer byte ordering is native ordering.

Reply:
We do or the data would not be read correctly.

Reply:
Right, just thinking it would be easier to debug/understand if this was explicit in the test rather than needing to recognize the data comparison failure is due to an endianness issue.  Easier when it's hex values, trickier otherwise.  Not must-fix.